todocore/src/test/java/com/knighttodo/todocore/integration/DayResourceIntegrationTest.java

```diff
@@ -220,4 +224,30 @@ public void deleteDay_shouldDeleteDayWithAllDayTodos_whenIdIsCorrect() throws Ex
         assertThat(dayRepository.findByIdAlive(day.getId())).isEmpty();
         assertThat(dayTodoRepository.findAllAlive().isEmpty());
     }
+
+    @Test
+    @Transactional
+    public void findDayByDate_shouldReturnExistingDay_whenDateIsCorrect() throws Exception {
+        Day day = dayRepository.save(DayFactory.dayInstance());
+        LocalDate date = LocalDate.now();
+        mockMvc.perform(get(buildGetDayByDate(date)))
+                .andExpect(status().isOk())
+                .andExpect(jsonPath(buildJsonPathToDate()).value(day.getDate().toString()));
+    }
+
+    @Test
+    public void findDayByDate_shouldRespondWithBadRequestStatus_whenWithoutDate() throws Exception {
+        String date = "";
+        mockMvc.perform(get(buildGetDayByDateWithoutDate(date)))
+                        .andExpect(status().is5xxServerError());
+        // This test has a bug, more details via jira code KNIG-13.
```

Comment:
it should be //todo this way it will be highlighted by Idea so it's easier to find

Reply:
Ok.